scripts/buildPublishedAPI.js

```diff
@@ -86,14 +96,25 @@ async function build() {
     }
   };
 
+  console.log('add packages to download from npm');
```

Comment:
remove or is this console status while running the add and not debugging? I saw the one below that made me think this wasn't debugging.

Reply:
all console logs are here as information on what is happening in the script, like status updates